ote/src/clj/ote/tasks/pre_notices.clj

```diff
@@ -125,45 +135,61 @@
+     table { border-collapse: collapse; mso-table-lspace: 0px; mso-table-rspace: 0px;}
+     td, a, span { border-collapse: collapse; mso-line-height-rule: exactly;}
+     .headerText1 {font-family:Roboto,helvetica neue,arial,sans-serif; font-size:2rem; font-weight:700;}
+     .headerText2 {font-family:Roboto,helvetica neue,arial,sans-serif; font-size:1.5rem; font-weight:700;}
      .whiteBackground {background-color:#FFFFFF}
      .grayBackground {background-color:#EFEFEF}
-     .btn {background-color:#0066CC;padding:15px;}
-     .footer {font-size:0.75rem}
+     .mcnButtonContent {background-color:#0066CC;padding:15px;}
+     .footer {font-family:Roboto,helvetica neue,arial,sans-serif;font-size:0.75rem;}
      a.mcnButton{display:block;}
      .even-row {background-color:#EFEFEF;}
      .odd-row {background-color:#FFFFFF;}
-      .tg  {border-collapse:collapse;border-spacing:0;}
-      .tg td{font-family:roboto,helvetica neue,arial,sans-serif;font-size:14px;padding:10px 5px;overflow:hidden;word-break:normal;}
-      .tg th{font-family:roboto,helvetica neue,arial,sans-serif;font-size:16px;font-weight:700;padding:10px 5px;overflow:hidden;word-break:normal;}
-      .tg .tg-oe15{background-color:#ffffff;text-align:left;vertical-align:top}
-      .tg .tg-lusz{background-color:#656565;color:#ffffff;text-align:left;vertical-align:top}
-      .tg .tg-vnjh{text-decoration:underline;background-color:#ffffff;color:#0066cc;text-align:left;vertical-align:top}
-      .tg .tg-m03x{text-decoration:underline;background-color:#efefef;color:#0066cc;text-align:left;vertical-align:top}
-      .tg .tg-fkgn{background-color:#efefef;border-color:#efefef;text-align:left;vertical-align:top}"]]
-   [:body.grayBackground
+     .tg  {border-collapse:collapse;border-spacing:0;}
+     .tg td{font-family:Roboto,helvetica neue,arial,sans-serif;font-size:14px;padding:10px 5px;overflow:hidden;word-break:normal;}
+     .tg th{font-family:Roboto,helvetica neue,arial,sans-serif;font-size:16px;font-weight:700;padding:10px 5px;overflow:hidden;word-break:normal;}
+     .tg .tg-oe15{background-color:#ffffff;text-align:left;vertical-align:top}
+     .tg .tg-lusz{background-color:#656565;color:#ffffff;text-align:left;vertical-align:top}
+     .tg .tg-vnjh{text-decoration:underline;background-color:#ffffff;color:#0066cc;text-align:left;vertical-align:top}
+     .tg .tg-m03x{text-decoration:underline;background-color:#efefef;color:#0066cc;text-align:left;vertical-align:top}
+     .tg .tg-fkgn{background-color:#efefef;border-color:#efefef;text-align:left;vertical-align:top}
+     @media only screen and (min-width:481px) and (max-width:599px) {
+      .headerText1 {font-family:Roboto,helvetica neue,arial,sans-serif; font-size:1.5rem; font-weight:700;}
+      .headerText2 {font-family:Roboto,helvetica neue,arial,sans-serif; font-size:1rem; font-weight:700;}
+     }"]]
+   [:body
     [:center
      [:div.whiteBackground {:width "80%" :style "padding-left: 20px; padding-right:20px;"}
```

Comment:
This padding-left and padding-right could be removed on the smaller breakpoint. Or atleast shrink the padding to at max 5px.